### PR TITLE
Migrates africas-talking.js and enables medic-collect.js

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

https://github.com/medic/angular10-migration/issues/78
https://github.com/medic/angular10-migration/issues/85

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
hey @newtewt 
Please have a look. Thanks for your time!

File: tests/e2e/africas-talking.js [hunk 1]
```diff
@@ -160,39 +160,37 @@ describe('africas talking api', () => {
       });
     };
 
-    beforeEach(done => {
-      submitSms({ from: '+64271234567', text: 'hello', id: 'a' })
-        .then(done)
-        .catch(done.fail);
+    beforeEach(() => {
+      return submitSms({ from: '+64271234567', text: 'hello', id: 'a' });
```
Comment (newtewt) [inline tests/e2e/africas-talking.js:164]: We can make this a nice one line by omitting the return.

`beforeEach(() => submitSms({ from: '+64271234567', text: 'hello', id: 'a' }));`

File: tests/e2e/africas-talking.js [hunk 2]
```diff
@@ -223,23 +222,18 @@ describe('africas talking api', () => {
...
-        .then(done)
-        .catch(done.fail);
+        });
     });
 
-    afterEach(done => {
-      utils.deleteDoc(savedDoc)
-        .then(done)
-        .catch(done.fail);
+    afterEach(() => {
```
Comment (newtewt) [inline tests/e2e/africas-talking.js:228]: Same as above, 

`afterEarch(() => utils.deleteDoc(savedDoc));`

**Review by newtewt — CHANGES_REQUESTED** [review]
Cool! Some minor changes about moving the Page objects.

File: tests/e2e/africas-talking.js [hunk 3]
```diff
@@ -160,39 +160,37 @@ describe('africas talking api', () => {
...
-    it('- shows content', () => {
-      commonElements.goToTasks();
-      commonElements.goToMessages();
+    it('- shows content', async () => {
+      await commonElements.goToTasks();
+      await commonElements.goToMessagesNative();
 
       // LHS
-      helper.waitElementToBeVisible(element(by.css('#message-list li:first-child')));
+      await helper.waitElementToBeVisibleNative(element(by.css('#message-list li:first-child')));
```
[the remark this answers is not in the document]
Comment (dianabarsan) [reply tests/e2e/africas-talking.js:172]: Yea, you've added a lot of these in your PR.
I think it's better to merge that one 1st and then see what's left to be added from this one. :)

**Comment by dianabarsan** [comment]
Could you have another look, @newtewt ? I was lucky to finally get no flakes and a green build!

**Review by newtewt — APPROVED** [review]
Nice! Feel free to merge.

**Pull request merged**